src/vs/workbench/services/suggest/browser/simpleSuggestWidget.ts

```diff
@@ -55,6 +55,7 @@ const enum WidgetPositionPreference {
 
 export const SimpleSuggestContext = {
 	HasFocusedSuggestion: new RawContextKey<boolean>('simpleSuggestWidgetHasFocusedSuggestion', false, localize('simpleSuggestWidgetHasFocusedSuggestion', "Whether any simple suggestion is focused")),
+	FocusedFirstSuggestion: new RawContextKey<boolean>('simpleSuggestWidgetFocusedFirstSuggestion', false, localize('simpleSuggestWidgetFocusedFirstSuggestion', "Whether the first simple suggestion is focused")),
```

Comment:
I think a "HasNavigated" context key would be more appropriate here, that way you could hit down and then up twice to go to the bottom of the list?

Reply:
I feel like changing the behavior based on user input will be confusing

Reply:
But we can go with that and see what ppl say